django_tables2/__init__.py

```diff
@@ -1,5 +1,5 @@
 # coding: utf-8
-from .tables import Table, TableBase
```

Comment:
the string 'table_factory' should be added to `__all__`, the `# noqa` comment should be obsolete then.